packages/expo-updates/android/src/main/java/expo/modules/updates/EnabledUpdatesController.kt

```diff
@@ -312,6 +323,20 @@ class EnabledUpdatesController(
     updatesConfiguration = UpdatesConfiguration.create(context, updatesConfiguration, configOverride)
   }
 
+  // UpdatesMetricsInterface implementations
+
+  override val runtimeVersion: String?
+    get() = updatesConfiguration.getRuntimeVersion()
```

Comment:
should this be `updatesConfiguration.runtimeVersionRaw` to match iOS (and what's returned in `getConstantsForModule`)

Reply:
Agreed